### ISSUE TITLE
Encoding will abort when using `--vpp-nnedi` together with `--vpp-denoise-dct` 

### ISSUE THREAD

**Issue by quamt** [rigaya/NVEnc#559]
Hello @rigaya,

when using `--vpp-nnedi` together with `--vpp-denoise-dct` the encoding will abort:
Using interlaced input material.

Error message:

```
cspconv: unsupported interlaced csp conversion: yv12(16bit) -> rgb(fp32).
denoise-dct: Error while running filter "cspconv".
denoise-dct: error at denoise: yv12(16bit).
Error while running filter "denoise-dct".
```

**Comment by rigaya** [comment]
Thank you for the report, I was able to reproduce the issue. I'll have this fixed in the next release.

**Comment by quamt** [comment]
Thank you.
I apologize for not mentioning earlier that when you mistakenly type the wrong number in `--vpp-denoise-dct` option, it displays incorrect option values in the error message.

Example:
```
Error: Invalid value "9" for "--vpp-denoise-dct step="
  Option value should be one of below...
    auto, fp32, fp16

Error: Invalid value "15" for "--vpp-denoise-dct block_size="
  Option value should be one of below...
    auto, fp32, fp16
```

**Comment by rigaya** [comment]
The vpp-nnedi issue and vpp-denoise-dct step and blocksize message should have been fixed in [NVEnc 7.42](https://github.com/rigaya/NVEnc/releases/tag/7.42).

**Comment by quamt** [comment]
Thank you very much for your excellent work.

When using the options `--vpp-nnedi` and `--vpp-denoise-dct` together, the encoding process works smoothly.
The error messages that display the option values are great as well.

```
Error: Invalid value "9" for "--vpp-denoise-dct step="
  Option value should be one of below...
    1, 2, 4, 8
```
    
```
Error: Invalid value "15" for "--vpp-denoise-dct block_size="
  Option value should be one of below...
    8, 16    
```

I am going to close this issue as the problem has been fixed.
